### ISSUE TITLE
Multilang build

### ISSUE THREAD

**Issue by dadaz** [wuxianlin/MIUI_patchrom_Find5#3]
Hi,

Could you build the Oneplus ROM in multilang ?

I have to use Jbart every updates to make it french. And I have to download full package every week, I can't use OTA.

Thanks for your great ROM. It's the best MIUI V6 Port for Oneplus One for me.

**Comment by wuxianlin** [comment]
I don't have mult languages resources.

**Comment by dadaz** [comment]
Too bad. I will ask X_S where he find multilanguages ressources and tell you if I can.

Thanks anyway.

**Comment by wuxianlin** [comment]
X-S's miui for OPO is based on CyanogenMod 11 while my miui for OPO is based on ColorOS.
CyanogenMod have mult languages.I will try to add some resources from Find7's International ColorOS.

**Comment by dadaz** [comment]
Yes I know but MIUI Languages ressources are only for MIUI, it doesn't use Android languages ressources as far as I know. That's why I have to use JBart to include language ressources. 

If you don't know JBart, it's a java based program which automatically translates MIUI (and some others) ROM. It uses miuiandroid.com language ressources.

Maybe you can use the same ressources, they have a lot of languages.

Maybe X_S takes internationnal version of MIUI as base, I don't know.

Thanks
